website/files/models/base.py

```diff
@@ -529,10 +529,15 @@ def touch(self, auth_header, revision=None, **kwargs):
         headers = {}
         if auth_header:
             headers['Authorization'] = auth_header
-        resp = requests.get(
-            self.generate_waterbutler_url(revision=revision, meta=True, **kwargs),
-            headers=headers,
-        )
+        try:
+            resp = requests.get(
+                self.generate_waterbutler_url(revision=revision, meta=True, **kwargs),
+                headers=headers,
+            )
+        except requests.ConnectionError:
```

Comment:
I'm not sure sure this is the right place to do this; `touch` is used in multiple places besides commenting. If it fails to reach Waterbutler, it should fail loudly most of the time. It might be better to catch this error only if we're trying to get an unread comment count?